zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/NotebookTest.java

```diff
@@ -1082,6 +1083,50 @@ public void testAuthorizationRoles() throws IOException {
     assertEquals(user2Notes.get(0).getId(), note.getId());
   }
 
+  @Test
+  public void testInterpreterSettingConfig() {
```

Comment:
Added test case. Can check the configuration changes after the interpreter type changes.